client/js/lounge.js

```diff
@@ -485,7 +490,14 @@ $(function() {
 		container.html(templates.user_filtered({matches: result})).show();
 	});
 
-	var forms = $("#sign-in, #connect, #change-password");
+	searchQuery.on("click", ".show-more-button", function() {
+		$(this).text("Loading older messages…").prop("disabled", true);
+		var query = searchQuery.find("form").data("lastQuery");
+		query.offset = searchQuery.find(".highlight").length;
+		socket.emit("search-query", query);
+	});
+
+	var forms = $("#sign-in, #connect, #change-password, #search-query");
```

Comment:
let

Reply:
This was here before my PR, please reach the initial author.